include/unifex/detail/unifex_fwd.hpp

```diff
@@ -24,7 +24,7 @@ namespace unifex {
     template <typename E, typename R, typename = void>
     extern const bool _can_execute;
 
-    template <typename Sender, typename Fn, typename = void>
+    template <typename S, typename F, typename = void>
```

Comment:
This trivial change is actualy necessary (!!!) because of https://developercommunity.visualstudio.com/content/problem/1042144/bug.html